.github/workflows/release.yml

```diff
@@ -24,7 +24,7 @@ jobs:
       - uses: ruby/setup-ruby@v1
         with:
           bundler-cache: true
-          ruby-version: '3.0'
+          ruby-version: '3.1'
```

Comment:
I believe this is required for us to use activesupport. The PR workflows pass because they are set to 3.1, but with the pre-release testing it failed on 3.0

Reply:
Got it!